src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -205,9 +216,11 @@ def aws_sign_in(aws_profile, duration_minutes=DEFAULT_SIGN_IN_DURATION_MINUTES,
             and _has_valid_session_credentials(aws_session_profile):
         return aws_session_profile
 
-    default_username = getpass.getuser()
+    default_username = get_default_username()
     username = input("Enter username associated with credentials [{}]: ".format(
         default_username)) or default_username
```

Comment:
This requires a tiny bit more user interaction than I think is necessary. Having `COMMCARE_CLOUD_DEFAULT_USERNAME` set in the environment indicates that it has been explicitly overridden. In that case it should not bother the user with this additional prompt.